### ISSUE TITLE
Can't Delete Tags in Actorinfo

### ISSUE THREAD

**Issue by natepley** [NiceneNerd/BCML#282]
Because of how BCML merges mods and vanilla files, tags I remove from actorinfo are automatically added back in when I install my mod with BCML.  I'm trying to make unsellable armors sellable for this mod - [https://gamebanana.com/mods/364976](url) .  I want to remove the CannotSell tag from these armors.

Would it be possible to make this work?

**Comment by ArchLeaders** [comment]
You need to remove said tag from the ActorLink file as well.

**Comment by natepley** [comment]
Yeah.  Removing it form the Actorpack works fine.  No issue there.  It's actorinfo that tags can't be removed from.  Unfortunately, changing actorinfo is necessary to make it actually sellable in the game.

**Comment by natepley** [comment]
> The actorinfo merger would require substantial revision to support tag deletions

Yeah.  I don't  know if you remember, but I talked to you about this awhile ago on Discord.  At the time, you said you might have had an idea to make it work, but then I never heard back from you.  I assume it just didn't work out.  If it can't be done, I can live without it.

**Comment by NiceneNerd** [comment]
Alas, I have retired from BCML except basic maintenance, so if it is to be implemented it will need to be by another contributor.
